src/briefcase/commands/config.py

```diff
@@ -0,0 +1,136 @@
+from pathlib import Path
+
+import tomli
+import tomli_w
+from platformdirs import PlatformDirs
+
+from briefcase.exceptions import BriefcaseConfigError
+
+from .base import BaseCommand
+
+
+class ConfigCommand(BaseCommand):
+    """Command to modify Briefcase configuration settings.
+
+    Allows setting of individual configuration keys within either a
+    project-level or global configuration file. Useful for scripting
+    or user-driven configuration outside of interactive prompts.
+    """
+
+    command = "config"
+    platform = None
+    output_format = None
+    description = (
+        "Set a configuration value for the current project or globally using the --global flag.\n\n"
+        "Configuration is stored in .briefcase/config.toml or the global config path.\n"
+        "Use keys like:\n"
+        "  - author.name\n"
+        "  - author.email\n"
+        "  - iOS.device\n"
+        "  - android.device\n"
+        "  - macOS.identity\n"
+        "  - network.proxy\n"
+        "  - network.cache_path\n\n"
+        "Precedence: CLI > project config > global config."
+    )
+    help = "Configure per-project or global settings."
```

Comment:
```suggestion
    description = "Set a configuration value that Briefcase will use as a default."
```
The `description` key is used for `briefcase --help` - it can't run to multiple lines; and there's no need for a separate `help` key.